### ISSUE TITLE
[BUG] Rich notification authorization request ended with error

### ISSUE THREAD

**Issue by mietzen** [IBM/mac-ibm-notifications#132]
**Describe the bug**

```
~/IBM\ Notifier.app/Contents/MacOS/IBM\ Notifier -type banner -title "Test title" -subtitle "Test subtitle" -main_button_label "Main button" -secondary_button_label "Secondary button" -tertiary_button_label "Tertiary button" -tertiary_button_cta_type link -tertiary_button_cta_payload "https://www.ibm.com"
Rich notification authorization request ended with error: Notifications are not allowed for this application
```

**To Reproduce**
Run IBM Notifier `banner` or `alert` on macOS 12.6 

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: macOS 12.6
 - Project version 2.8.0

**Comment by SMartorelli** [comment]
Hi, does this happen on the first trigger of a banner/alert after IBM Notifier installation?

**Comment by mietzen** [comment]
Yes, there was also no "alert" from the notification center asking for allowance to display notifications.

**Comment by mietzen** [comment]
Ok, found it: `spctl --add ~/IBM\ Notifier.app/Contents/MacOS/IBM\ Notifier` works.

**Comment by SMartorelli** [comment]
This is a known bug that doesn't really depends from IBM Notifier but from how macOS handle new notifications payload. Most of the times a reboot works and allow the system to "load" the new settings. If `spctl --add ~/IBM\ Notifier.app/Contents/MacOS/IBM\ Notifier` works for you that's a news also for us, we'll verify if we can suggest this as common workaround for the issue. Thanks!

**Comment by mietzen** [comment]
> This is a known bug that doesn't really depends from IBM Notifier but from how macOS handle new notifications payload. Most of the times a reboot works and allow the system to "load" the new settings. If `spctl --add ~/IBM\ Notifier.app/Contents/MacOS/IBM\ Notifier` works for you that's a news also for us, we'll verify if we can suggest this as common workaround for the issue. Thanks!

Just a Note: Rebooting didn't work in my case.

**Comment by SMartorelli** [comment]
@mietzen please check the solution found for this annoying issue https://github.com/IBM/mac-ibm-notifications/wiki/Known-Issues-And-Fixes#4-notification-agent-rich-notification-authorization-request-ended-with-error-notifications-are-not-allowed-for-this-application
